spec/mocha.helpers.js

```diff
@@ -69,7 +69,7 @@ const getUserDefinedWindowProperties = () => {
   document.body.appendChild(iframe);
 
   const currentWindow = Object.getOwnPropertyNames(window);
-  const properties = currentWindow.filter((prop) => !Object.prototype.hasOwnProperty.call(iframe.contentWindow, prop));
+  const properties = currentWindow.filter(prop => !Object.prototype.hasOwnProperty.call(iframe.contentWindow, prop));
```

Comment:
Is there any chance your eslint version is out of sync? I'm getting an error with this new line and I think this line was specifically changed in the eslint update PR: https://github.com/Constructor-io/constructorio-client-javascript/pull/190

Reply:
Hmm I think I dind't run npm install so I will redo these lint changes.